server/store/datastore/migration/031_unify_columns_tables.go

```diff
@@ -0,0 +1,642 @@
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_pid", "pid"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_ppid", "ppid"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_name", "name"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_state", "state"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_error", "error"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_failure", "failure"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_exit_code", "exit_code"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_started", "started"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_stopped", "stopped"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "steps", "step_type", "type"); err != nil {
+			return err
+		}
+
+		// Task
+		if err := renameColumn(sess, "tasks", "task_id", "id"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "tasks", "task_data", "data"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "tasks", "task_labels", "labels"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "tasks", "task_dependencies", "dependencies"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "tasks", "task_run_on", "run_on"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "tasks", "task_dep_status", "dep_status"); err != nil {
+			return err
+		}
+
+		// User
+		if err := renameColumn(sess, "users", "user_id", "id"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_login", "login"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_token", "token"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_secret", "secret"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_expiry", "expiry"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_email", "email"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_avatar", "avatar"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_admin", "admin"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_hash", "hash"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "users", "user_org_id", "org_id"); err != nil {
+			return err
+		}
+
+		// Workflow
+		if err := renameColumn(sess, "workflows", "workflow_id", "id"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "workflows", "workflow_pipeline_id", "pipeline_id"); err != nil {
+			return err
+		}
+		if err := renameColumn(sess, "workflows", "workflow_pid", "pid"); err != nil {
```

Comment:
`parent_id`?

Reply:
You are right `ppid` is the parent. Not sure if `process id` /  `number` would be a real improvement. We can leave it for now.